pages/en/participate/reporting.mdx

```diff
@@ -38,4 +38,4 @@ Please avoid sending issues via Discord DMs. Instead, try following one of the o
 
 ### Join the security discussion
 
-Thanks for your help in securing the network! If you would like to learn more and discuss security topics related to Mina, we invite you to participate in [the #security channel on Mina’s Discord](https://discord.gg/T3raKufc).
+Thanks for your help in securing the network! If you would like to learn more and discuss security topics related to Mina, we invite you to participate in the [#security channel](https://discord.gg/T3raKufc) on Mina’s Discord.
```

Comment:
`https://discord.gg/T3raKufc` seems to be invalid now :(

Reply:
Thanks! Fixed :D How do you find these invalid links so quickly? I think we might a few more in the docs.